### PR TITLE
FUSETOOLS-3257 - adding new 7.3.1 release version and catalog cache b…

### PR DESCRIPTION
**Pull request by lhein**
…undle

Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
 	public static final String CAMEL_SPRING_BOOT_STARTER = "camel-spring-boot-starter";
 	public static final String CAMEL_WILDFLY = "org.wildfly.camel";
 	
-	public static final String CAMEL_VERSION_LATEST_COMMUNITY = "2.23.1";
+	public static final String CAMEL_VERSION_LATEST_COMMUNITY = "2.24.1";
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java]: this is a separated goal than the 7.3.1 upgrade, please do it with a separate task and PR

Comment (lhein) [reply core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java]: reverted

**Comment by lhein** [comment]
> I think that several test Manifest files needs to be updated, such as
> 
> https://github.com/jbosstools/jbosstools-fuse/blob/f2c27181be1ae69f6d3cd17df2542c0827a57528/editor/tests/org.fusesource.ide.projecttemplates.tests.integration/META-INF/MANIFEST.MF#L52

Actually only 2 Manifest files contained that bundle. 
org.fusesource.ide.camel.editor.tests.integration
org.fusesource.ide.projecttemplates.tests.integration

File: core/plugins/org.fusesource.ide.camel.model.service.impl/src/org/fusesource/ide/camel/model/service/internal/CamelModelPatcher.java [hunk 2]
```diff
@@ -164,6 +164,9 @@ private static void applyFixesToComponentsSyntax(CamelModel loadedModel) {
 		applyFixToComponent(loadedModel, "micrometer", "micrometer:metricsType:meterName", "micrometer:metricsType:metricsName");
 		applyFixToComponent(loadedModel, "service", "service:serviceName:delegateUri", "service:delegateUri");
 		applyFixToComponent(loadedModel, "nsq", "nsq:lookupServer", "nsq:servers");
+		applyFixToComponent(loadedModel, "corda", "corda:url", "corda:configuration");
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.camel.model.service.impl/src/org/fusesource/ide/camel/model/service/internal/CamelModelPatcher.java]: thsi is not related to 7.3.1 version but to 2.24 Camel catalog. Would be really best to have it in a different commit/PR

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
all tests passed, builds green, ready for another review

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Pull request merged**